rfcs/text/0017/exclude-set.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Looks like you have some good test case built! These will be good examples to help build out some of the unit testing.

Reply:
thx!